frontend/src/components/BookingModal.vue

```diff
@@ -63,15 +66,19 @@
       </div>
       <div class="mx-auto flex w-4/5 items-stretch justify-center gap-8">
         <secondary-button
+          class="mdl-btn-close"
           :label="t('label.close')"
           @click="emit('close')"
+          :title="t('label.close')"
         />
         <primary-button
           v-if="!isFinished"
+          class="btn-book"
```

Comment:
Should we split off the modal classes even more? Or maybe fold the mdl-btn-close to just btn-close?

Reply:
Good question, I changed all `.mdl-btn-close` to just `.btn-close` for now. If we need something different, we can always just change it.